COMPILING.md

```diff
@@ -75,6 +69,12 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
    make -C src IPASIR=../../ipasir LIBSOLVER=$(pwd)/ipasir/libipasir.a
    ```
 
+5. To compile JBMC, do
+   ```
+   make -C cbmc-git/jbmc/src java-models-library-download
```

Comment:
minisat2 will also be needed

Reply:
java-models-library will need versioning. I could either add a submodule or specify a tagged version to download.

Reply:
@Degiorgio might be working on that? (https://github.com/diffblue/cbmc/pull/2283#issuecomment-395056049)

Reply:
I haven't started working on this, but this will definitely be needed. I am not sure if there is any license related issues with adding it as a submodule and hence I was going to go for  tagged versions.

Reply:
I haven't given any thought to licensing, but I think tagging is preferable either way as it enforces good citizenship on versioning. JBMC might go as far as printing the version of the models library on start-up, but that's for JBMC leads to consider.